src/Transformers/JsonApi/resource/normalizeResources.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @param {Array} included
+ *
+ * @return {Map}
+ */
+import normalizeResource from './normalizeResource';
+
+/**
+ * @param {Array} items
+ *
+ * @returns {Object}
+ */
+export default function normalizeResources(items) {
+  const normalizedItems = {};
+
+  items.forEach((item) => {
+    normalizedItems[`${item.type}-${item.id}`] = normalizeResource(item);
```

Comment:
Wat is de reden dat je de type voor de id zet? Is er een kans dat er objecten van meer dan 1 type in de response zitten?

Reply:
Zeker, in de included kunnen allerlei types zitten en als je geen uuid's hebt is de kans zelfs groot dat de id's gelijk zijn. Vandaar deze aanpak, zo is de key gegarandeerd uniek.